lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -38,7 +38,7 @@ def test_copy_tar(self):
             self.config,
             self.logger,
         )
-        crt.copy_result_tb("token")
+        crt.copy_result_tb("token", "access")
```

Comment:
Give my other comments about an optional parameter, I think we'll want to validate the behavior with and without the access parameter.

Reply:
ok.. So, we will move away from `default` option in click.

Reply:
Re: "move away from `default` option in click": In terms of server behavior, the `?access` is optional, and there's a default.

However, *always* supplying this at the agent side *decouples* the agent and server default behaviors, which is not a bad thing. Regardless of the fact that the server (currently) defaults to `"private"`, this may change when we add groups: for example, we could even configure things so that the server's default behavior would be access for the user's designated "main" group. And having a decoupled Agent default (e.g., `"private"`) means that doesn't immediately impact Agent behavior. (We don't necessarily have to synchronize an Agent release.)

Regardless, unit tests should validate `--access=private`, `--access-public` and absent `--access` as well as `--access=bad-value`.